app/models/column/positioned.rb

```diff
@@ -5,6 +5,8 @@ module Column::Positioned
     scope :sorted, -> { order(position: :asc) }
 
     before_create :set_position
+    after_create_commit -> { surroundings.touch_all }
+    after_destroy_commit -> { surroundings.touch_all }
```

Comment:
Do we need this explicit invalidation? In theory we consider adjacent columns for caching here: https://github.com/basecamp/fizzy/blob/ca10e4ba4e7e122580d59a7b1135c058e13a9dfc/app/views/boards/show/_columns.html.erb#L29